jwt/main.go

```diff
@@ -57,7 +63,8 @@ func main() {
 		Expiry:    expiry,
 	}
 
-	customCl := customClaims{"degenerates", "b6123f22-79b6-11ea-8bde-c77510c60c52"}
+	customCl := customClaims{"degenerates", "b6123f22-79b6-11ea-8bde-c77510c60c52",
+		RateLimitType{5, "SECOND"}}
```

Comment:
Can you make the number a command line parameter?

Reply:
Done